File: Arrays 101/01 What is an Array/Max Consecutive Ones.js
class Solution {
    findMaxConsecutiveOnes(nums) {
        let count = 0;
        let maxCount = 0;

        for (let num of nums) {
            if (num !== 1) {
                maxCount = Math.max(maxCount, count);
                count = 0;
            } else {
                count++;
            }
        }

        return Math.max(maxCount, count);
    }
}

File: Arrays 101/01 What is an Array/Max Consecutive Ones.cs
public class Solution 
{
    public int FindMaxConsecutiveOnes(int[] nums) 
    {
        int count = 0;
        int maxCount = 0;

        foreach (int num in nums)
        {
            if (num != 1)
            {
                maxCount = Math.Max(maxCount, count);
                count = 0;
            }
            else
            {
                count++;
            }
        }

        return Math.Max(maxCount, count);
    }
}